src/pager.rs

```diff
@@ -125,6 +156,10 @@ impl Pager {
 
     /// Display a temporary message at the prompt area
     ///
+    /// A message is a piece of text that the main application can ask minus to display at the
+    /// prompt location. The text message is temporary and will get cleared whenever the use
+    /// rdoes a action on the terminal like pressing a key or scrolling using the mouse.
+    ///
```

Comment:
this isn't needed as the first line already includes enough information in my opinion.

Reply:
Maybe we can trim it down a little bit to only mention that any key/mouse action will remove that message.